docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,534 @@
+
+Besides the main namespaces, add `failover` namespaces, in both clusters, in case of a total region loss. This is for completeness, so you don't forget to add the mapping on region recovery. The operational procedure is handled in a different [document](#). <!-- TOOD: add reference -->
+
+1. Configure CoreDNS to send requests for certain namespaces to the remote DNS resolver.
+
+```bash
+kubectl --context $CLUSTER_0 edit -n kube-system configmap coredns
+```
+
+At the bottom of the `Corefile`, add the namespace mapping to the remote DNS resolver:
+
+<!-- yaml is always reformated wrongly -->
+
+:::info
+Make sure to change the namespaces `camunda-paris` and `camunda-paris-failover` according to your chosen namespaces. Additionally, make sure to change the IP addresses to the ones you've previously retrieved via the AWS CLI.
+
+The following are just example values and will not work in your setup!
+:::
+
+```bash
+# Example - You are configuring the CoreDNS of the london cluster
+# to redirect any DNS requests for camunda-paris to the paris cluster
+camunda-paris.svc.cluster.local:53 {
+  errors
+  cache 30
+  forward . 10.202.30.193 10.202.46.72 10.202.65.75 {
+    force_tcp
+  }
+}
+camunda-paris-failover.svc.cluster.local:53 {
+  errors
+  cache 30
+  forward . 10.202.30.193 10.202.46.72 10.202.65.75 {
+    force_tcp
+  }
+}
+```
+
+<details>
+  <summary>Full example</summary>
+  <summary>
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    eks.amazonaws.com/component: coredns
+    k8s-app: kube-dns
+  name: coredns
+  namespace: kube-system
+data:
+  Corefile: |
+    .:53 {
+        errors
+        health {
+            lameduck 5s
+          }
+        ready
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
+          pods insecure
+          fallthrough in-addr.arpa ip6.arpa
+        }
+        prometheus :9153
+        forward . /etc/resolv.conf
+        cache 30
+        loop
+        reload
+        loadbalance
+    }
+    camunda-paris.svc.cluster.local:53 {
+        errors
+        cache 30
+        forward . 10.202.30.193 10.202.46.72 10.202.65.75 {
+            force_tcp
+        }
+    }
+    camunda-paris-failover.svc.cluster.local:53 {
+        errors
+        cache 30
+        forward . 10.202.30.193 10.202.46.72 10.202.65.75 {
+            force_tcp
+        }
+    }
+```
+
+  </summary>
+</details>
+
+2. Repeat **Step 1** for the Paris cluster to configure the CoreDNS of Paris to redirect any DNS requests to the London DNS resolver for your chosen namespace, like `camunda-london`.
+
+3. Check that CoreDNS has reloaded for the changes to take effect before continuing. Make sure it contains `Reloading complete`:
+
+```bash
+kubectl --context $CLUSTER_0 logs -f deployment/coredns -n kube-system
+kubectl --context $CLUSTER_1 logs -f deployment/coredns -n kube-system
+```
+
+</TabItem>
+<TabItem value="test-dns-chaining" label="(optional) Test DNS chaining">
```

Comment:
let's drop `optional` here, I think it would be better the users check their setup always, as early in the process as possible.